src/components/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Good work @codjoero.
I see the Header component is indirectly in the components folder.
Don't you think it would be better off if we categorize components in terms of what they are representing?
In this case, I'd rather the Header component to be under a sub-folder. e.g **src/components/UI/Header.jsx** because a header component is more of a UI component.